### PR TITLE
login session management

### PR DESCRIPTION
**Pull request by Geekiter**

### CODE REVIEW

File: src/dictionaries/es-ES.json [hunk 1]
```diff
@@ -66,5 +66,8 @@
       "successToast": "Location created successfully",
       "errorToast": "Oops, Error creating location"
     }
+  },
+  "login-session": {
+    "menu": "Yours Devices"
```
Comment (Poordeveloper) [inline src/dictionaries/es-ES.json:71]: change to `Login Sessions`

File: prisma/schema.prisma [hunk 2]
```diff
@@ -51,6 +52,7 @@ model User {
   accounts      Account[]
   sessions      Session[]
   organizations Organization[]
+  LoginSession  LoginSession[]
```
Comment (Poordeveloper) [inline prisma/schema.prisma:55]: loginSessions, don't create your own style please.

File: prisma/schema.prisma [hunk 3]
```diff
@@ -280,3 +282,20 @@ model Settings {
...
+  session   Session  @relation(fields: [sessionId], references: [id], onDelete: Cascade)
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  ip        String
+  country   String
+  region    String
+  city      String
+  os        String
+  browser   String
+  userAgent String
```
Comment (Poordeveloper) [inline prisma/schema.prisma:300]: put `ip` -> `userAgent` into one json field please.

File: src/app/[locale]/(mainboard)/login-session/_components/logout.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  sessionId: string;
+}
+function Logout({ sessionId }: Props) {
+  const delete_devices = async (sessionId: string) => {
+    const response = await fetch("/api/device_logout", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ sessionId }),
```
Comment (Poordeveloper) [inline src/app/[locale]/(mainboard)/login-session/_components/logout.tsx:15]: Wierd

**Review by Poordeveloper — CHANGES_REQUESTED** [review]

**Review by mocusez — CHANGES_REQUESTED** [review]

File: src/app/[locale]/(mainboard)/dashboard/page.tsx [hunk 5]
```diff
@@ -12,7 +13,10 @@ type Props = {
 };
 
 export default async function DashBoard({ params: { locale } }: Props) {
-  const session = await getServerAuthSession();
+  const headersList = headers();
+  const user_agent = headersList.get("user-agent");
+  const user_ip = headersList.get("x-forwarded-for");
+  const session = await getServerAuthSession(user_agent ?? "", user_ip ?? "");
```
Comment (Poordeveloper) [inline src/app/[locale]/(mainboard)/dashboard/page.tsx:19]: Why getting user_ip is associated with dashboard page?

**Pull request closed without merging**